File: sdkwork-app-sdk-typescript/src/types/vip-daily-reward-vo.ts
/** VIP每日奖励响应 */
export interface VipDailyRewardVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 奖励灵点数 */
  rewardPoints?: number;
  /** 领取时间 */
  claimedAt?: string;
  /** 消息 */
  message?: string;
  /** 连续签到天数 */
  consecutiveDays?: number;
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/vip_daily_reward_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class VipDailyRewardVO:
    """VIP每日奖励响应"""
    created_at: str = None
    updated_at: str = None
    reward_points: int = None
    claimed_at: str = None
    message: str = None
    consecutive_days: int = None
